.gitignore

```diff
@@ -81,3 +81,8 @@ perf.data*
 # copyrighted assets
 /assets/converted/
 /assets/terrain/
+
+# downloaded assets
+download/*
+nyan
+package/*
```

Comment:
Do these folders have to be in the root folder or can they be in `build`? Then we don't need extra ignore rules for them.

Reply:
yes he can be move in build I just copy paste CI